eldritch/mythos.py

```diff
@@ -456,13 +665,176 @@ def get_trigger(self, event, state):
     return super().get_trigger(event, state)
 
 
+class Mythos28(Headline):
+  def __init__(self):
+    super().__init__("Mythos28", "Graveyard", "Isle", {"hex"}, {"slash", "triangle", "star"})
+
+  def create_event(self, state):
+    first_player = state.characters[state.first_player]
+    seq = super().create_event(state)
+    if not first_player.gone:
+      check = events.Check(first_player, "luck", -1)
+      pass_fail = events.PassFail(first_player, check, events.Nothing(), events.Curse(first_player))
+      seq.events.extend([check, pass_fail])
```

Comment:
No need to put both the check and the pass_fail in here - PassFail will automatically return a sequence if argument 2 is an event (as opposed to a value).